wlauto/workloads/multiapp/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java

```diff
@@ -90,6 +91,8 @@ private void logIntoSkype()  throws Exception {
         shareUsingApp("Skype");
         skype.handleLoginScreen(loginName, loginPass);
         confirmAccess();
+
+        sleep(10); // Pause while the app settles before returning
```

Comment:
In reality it does't appear to take 10 seconds as other processes are in action while this sleep occurs. It was reliable at 10 seconds. I can test with smaller numbers but all this testing takes time. Maybe 30 minutes per change.